docs/index.md

```diff
@@ -59,7 +59,7 @@ pip install lumapps-sdk
 
     ```python
     user_email = "YOUR EMAIL"
-    usr = client.get_call("user/get", email=user_email)
+    usr = api.get_call("user/get", email=user_email)
```

Comment:
TO fix this you have to modify the README.md and then regenerate the doc (the index.md is a generated file based on the README).